packages/builder/src/stores/builder/users.ts

```diff
@@ -0,0 +1,72 @@
+import { get, derived } from "svelte/store"
+import { BudiStore } from "../BudiStore"
+import { UIUser } from "@budibase/types"
+
+export class UserStore extends BudiStore<UIUser[]> {
+  actions: {
+    init: (users: UIUser[]) => void
+    updateUser: (user: UIUser) => void
+    removeUser: (sessionId: string) => void
+    reset: () => void
+  }
+
+  constructor() {
+    super([])
+    this.actions = {
+      init: this.init.bind(this),
+      updateUser: this.updateUser.bind(this),
+      removeUser: this.removeUser.bind(this),
+      reset: this.reset.bind(this),
+    }
+  }
+
+  init(users: UIUser[]) {
+    this.store.set(users)
```

Comment:
NAB: BudiStore already binds `set`, `subscribe` and `update` to itself, so you can just do `this.update`, `this.set` etc. Again, no functional difference though!